### DISCUSSION TITLE
Search repos with "string1" in one file and "string2" in another file

### DISCUSSION THREAD

**Discussion by mikelange49** [community/community#13039] [Other Feature Feedback, Questions, & Ideas]
and possibly expand to :
Search organizations with "string1" in one file of one repo, and "string2" in another file of another repo

**Comment by StingyJack** [comment]
Upvoted for the title idea. Ive had to locate a correct repo within my org before and had to write scripts to join two results like this. 

The string 1 in one repo and string 2 in a different repo sounds like two searches to me. Not sure how that would be useful if combined.

**Comment by lobsterkatie** [comment]
I'd also like this. I've tried using a boolean operator but it doesn't seem to work:

`(path:package.json content:/"@sentry\/nextjs": "6/) AND (path:next.config.js content:withSentryConfig)`

(Seems like this is related: https://github.com/github/feedback/discussions/12834)

**Reply by look** [reply]
This search isn't satisfiable because you're requesting results in `package.json` AND `next.config.js`. You could OR the conditions and restrict by owner or repository. I don't think that's what you want, but might work for some situations.

**Reply by lobsterkatie** [reply]
Huh. Why is that not "I want a repo where path A has content A and path B has content B?"

**Reply by look** [reply]
@lobsterkatie Currently, search queries are evaluated on a per-document basis and return documents. So we aren't able to express aggregation queries like that yet.
